pkg/kubectl/validation/schema.go

```diff
@@ -20,35 +20,11 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
-	"reflect"
-	"regexp"
-	"strings"
 
-	swagger "github.com/emicklei/go-restful-swagger12"
 	ejson "github.com/exponent-io/jsonpath"
-	"github.com/golang/glog"
-	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
-	"k8s.io/apimachinery/pkg/util/yaml"
-	apiutil "k8s.io/kubernetes/pkg/api/util"
 )
 
-// InvalidTypeError records information about an invalid type.
-type InvalidTypeError struct {
```

Comment:
A little bit surprised that we have an error type that's only used by the swagger.

Reply:
This are just convenience type used by Swagger 1.2. OpenAPi validation has redefined these types ...